include/dxc/Support/HLSLOptions.h

```diff
@@ -187,6 +187,7 @@ class DxcOpts {
   bool ResMayAlias = false; // OPT_res_may_alias
   unsigned long ValVerMajor = UINT_MAX, ValVerMinor = UINT_MAX; // OPT_validator_version
   unsigned ScanLimit = 0; // OPT_memdep_block_scan_limit
+  unsigned long DisabledOptimizations; // OPT_opt_disable
```

Comment:
If the suggested struct is used, that can simply be used here as well, eliminating a location where each would have to be spelled out twice.